ecs-agent/netlib/platform/common_linux.go

```diff
@@ -182,6 +183,12 @@ func (c *common) buildAWSVPCNetworkNamespaces(
 	if err != nil {
 		return nil, err
 	}
+
+	logger.Info("Building network configuration for awsvpc task", map[string]interface{}{
```

Comment:
nit: prefer `logger.Fields` to a map literal, and we will usually add commonly used fields like the one you are introducing to the constants file [[link](https://github.com/aws/amazon-ecs-agent/blob/master/ecs-agent/logger/field/constants.go)]

Same comment for all of these logs